src/support/export-csv.worker.ts

```
@@ -0,0 +1,220 @@
+import { Collection } from "collect.js";
+import { safe } from "./requests";
+import { Item } from "./requests/records/item";
+import { TriggeredBy, WorkbookJobWorker } from "./utils/job.worker";
+import api from "@flatfile/api";
+import path from "path";
+import os from "os";
+import fs from "fs";
+
+@TriggeredBy("export-csv")
+export class ExportCsvWorker extends WorkbookJobWorker {
+  async execute(): Promise<void | JobOutcome> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)